lib/generators/clearance/install/install_generator.rb

```diff
@@ -20,7 +20,10 @@ def inject_clearance_into_application_controller
       end
 
       def create_or_inject_clearance_into_user_model
-        if File.exist? "app/models/user.rb"
+        if (File.exist? "app/models/user.rb") &&
+           (File.readlines("app/models/user.rb")
```

Comment:
Maybe extract the content check part of this to a separate well named method - `user_model_includes_clearance?` (or something)?